lib/physiqual/data_services/summarized_data_service.rb

```diff
@@ -112,6 +110,10 @@ def cluster_in_buckets(data, from, to)
           # TODO: Just flatten whatever you add before adding it and check that your concat works correctly.
           buckets[current_bucket][values_field] = buckets[current_bucket][values_field].flatten
         end
+        # remove the extra information
+        buckets.each_with_index do |_bucket, bucket_index|
+          buckets[bucket_index].delete(date_time_start_field)
```

Comment:
yeah ik dacht dat we dit misschien nog wel toe zouden kunnen voegen, het maakt het heel concreet hoe groot je bucket is.